main.go

```diff
@@ -834,13 +840,7 @@ func mainCore() error {
 		wg.Wait()
```

Comment:
Double oops.  The call to `RegisterNodeNtfnHandlers` should be fine after starting the monitors as long as it is before `wg.Wait()`.  Anyway, we've always previously called `RegisterNodeNtfnHandlers` before starting the monitors, so I don't think it's an issue as long as these things are done back-to-back.

Reply:
I've revised the PR title to reflect this.